src/ImageSharp/Formats/Jpeg/GolangPort/OrigJpegConstants.cs

```diff
@@ -227,6 +209,32 @@ internal static class Markers
         }
 
         /// <summary>
+        /// Contains JFIF specific markers
+        /// </summary>
+        public static class JFif
```

Comment:
is this still needed now you've moved to the `ToAsciiByte` style?

Reply:
No. Good spot. I'll delete it, thought I already had.